recipe/build.sh

```diff
@@ -42,14 +42,16 @@ function bootstrap_build {
 # https://www.erlang.org/doc/installation_guide/install-cross#Build-and-Install-Procedure_Building-With-configuremake-Directly_Building-a-Bootstrap-System
 if [[ "${CONDA_BUILD_CROSS_COMPILATION}" -eq 1 ]]; then
   bootstrap_build
+  # erl_xcomp_sysroot is needed for cross-compilation to find SSH headers for target arch.
+  # https://github.com/erlang/otp/blob/master/HOWTO/INSTALL-CROSS.md#cross-system-root-locations
+  export erl_xcomp_sysroot="${CONDA_BUILD_SYSROOT}"
```

Comment:
Do we know what headers it is looking for?

Reply:
I don't know unfortunately. I am planning to look into it more at some point, but for now my goal is to be guided by the fact that RabbitMQ is working. :-)

Reply:
That's great news! 🙂

In that case let's go ahead and add this to `main`

Also would be good to file an issue for follow up

Reply:
I am planning a bit more tests with RMQ once I hear back from its maintainer and we'll publish the package. So I'd hold off on that until then if you don't mind. (the version on main is anyway not useful for RabbitMQ yet)

Reply:
> Also would be good to file an issue for follow up

Yep, I'll create an issue

Reply:
Thanks Daniel! 🙏

Am ok either way

Would there be an issue for users installing Erlang latest without this patch (outside of RabbitMQ)?

Reply:
In principle yes, if they needed the SSH/crypto module functionality. In other words, the current build is not broken, but is just lacking these optional modules. But in that case they will get a useful and meaningful error right at the application startup so to me that's not a huge issue (especially given that the ARM build is literally few weeks old).

But let's say that I'll target merging to main by the end of next week regardless.

Reply:
Sounds good. Happy to wait until end of next week 🙂